src/extension.ts

```diff
@@ -31,6 +31,9 @@ export function getFormattedText(
     if (config.keywordCase != null) {
       formattingOptions.keywordCase = CaseOptionEnum[<keyof typeof CaseOptionEnum>config.keywordCase];
     }
+    if (config.datatypeCase != null) {
+      formattingOptions.keywordCase = CaseOptionEnum[<keyof typeof CaseOptionEnum>config.datatypeCase];
```

Comment:
Have to tried it with this change?  I doubt it would work since you're just overriding the `keywordCase` option, which passes `--keyword-case` to pgFormatter.  But, it looks like you want `--data-type` option to be passed.

Reply:
Yeah you right. Need use type-case
```
-U | --type-case N    : Change the case of the data type name. Default is
                                lowercase: 1. Values: 0=>unchanged, 1=>lowercase,
                                2=>uppercase, 3=>capitalize.
```

Reply:
Sorry i undestand what you mean 
You use  psqlformat(https://github.com/bradymholt/psqlformat) but in that lib not defined type-case

Reply:
<img width="472" alt="image" src="https://github.com/bradymholt/vscode-pgFormatter/assets/11075696/dd15452e-2e63-42f0-be09-348da59a6c6d">

Reply:
Maybe you can provide?

Reply:
Maybe I can provide what?>

Reply:
Implement type-case 
from https://github.com/darold/pgFormatter
for https://github.com/bradymholt/psqlformat#readme